### ISSUE TITLE
Nuxt 3.14 - Duplicated imports "getPathRobotConfig"

### ISSUE THREAD

**Issue by ronenteva** [harlan-zw/nuxt-seo#347]
### Describe the bug

After upgrading to Nuxt 3.14, I keep getting the following warning (using nuxt-seo v2):
```
Duplicated imports "getPathRobotConfig", the one from ".../node_modules/.pnpm/@nuxtjs+sitemap@6.1.4_h3@1.13.0_magicast@0.3.5_rollup@4.24.4_vite@5.4.10_@
types+node@22.9.0_s_ntq346bkr5yndz4g4k4j5oxh3q/node_modules/@nuxtjs/sitemap/dist/runtime/nitro/composables/getPathRobotConfigPolyfill" has been ignored and ".../node_module
s/.pnpm/@nuxtjs+robots@4.1.11_magicast@0.3.5_rollup@4.24.4_vite@5.4.10_@types+node@22.9.0_sass-embedd_5k5rijypjtwsugxtx3jxakcgzu/node_modules/@nuxtjs/robots/dist/runtime/nitro/composables/getPathRobotConfig.js" is used
```

### System / Nuxt Info

```Shell
- Operating System: Linux
- Node Version:     v20.18.0
- Nuxt Version:     3.14.0
- CLI Version:      3.15.0
- Nitro Version:    2.10.2
- Package Manager:  pnpm@9.12.3
- Builder:          -
- User Config:      default
- Runtime Modules:  @nuxt/scripts@0.9.5, @nuxt/ui, @nuxtjs/html-validator@1.8.2, @nuxtjs/i18n@8.5.6, @nuxtjs/seo@2.0.0-rc.23, @pinia/nuxt@0.7.0, @vueuse/nuxt@11.2.0, nuxt-security@2.0.0, nuxt-time@1.0.2
- Build Modules:    -
```

**Comment by harlan-zw** [comment]
Hi, I believe this is fixed with the latest robots version, if you do a  `nuxi upgrade --force` it might solve it.

**Comment by XStarlink** [comment]
Thanks @harlan-zw!
I had the same problem as @ronenteva but as you suggested, using `npx nuxi upgrade --force` worked for me !

Fixed with `nuxt@3.14.159`

**Comment by ronenteva** [comment]
Yes, seems like all good now. 
Thanks!
